elastalert/loaders.py

```diff
@@ -93,7 +93,8 @@ class RulesLoader(object):
         'metric_aggregation': ruletypes.MetricAggregationRule,
         'percentage_match': ruletypes.PercentageMatchRule,
         'spike_aggregation': ruletypes.SpikeMetricAggregationRule,
-        'error_rate': ruletypes.ErrorRateRule  #Adding Error Rate Rule type
+        'error_rate': ruletypes.ErrorRateRule,  #Adding Error Rate Rule type
+        'advance_search': ruletypes.AdvanceSearchRule
```

Comment:
no better name than this?

Reply:
If no better name, at least change it to `advanced_query`? We exposed this feature as advanced query only?